scripts/fuzz_opt.py

```diff
@@ -159,6 +169,80 @@ def randomize_fuzz_settings():
     print('randomized settings (NaNs, OOB, legalize, extra V8_OPTS):', NANS, OOB, LEGALIZE, extra_v8_opts)
 
 
+def pick_initial_contents():
+    # whether we use an initial wasm file's contents as the basis for the
+    # fuzzing, or if we start entirely from scratch
+    global INITIAL_CONTENTS
+
+    INITIAL_CONTENTS = None
+    # half the time don't use any initial contents
+    if random.random() < 0.5:
+        return
+    test_name = random.choice(all_tests)
+    print('initial contents:', test_name)
+    assert os.path.exists(test_name)
+    # tests that check validation errors are not helpful for us
+    if '.fail.' in test_name:
+        print('initial contents is just a .fail test')
+        return
+    if os.path.basename(test_name) in [
+        # contains too many segments to run in a wasm VM
+        'limit-segments_disable-bulk-memory.wast',
+        # https://github.com/WebAssembly/binaryen/issues/3203
+        'simd.wast',
+        # corner cases of escaping of names is not interesting
+        'names.wast',
+        # huge amount of locals that make it extremely slow
+        'too_much_for_liveness.wasm',
+        # these contain illegal pops()
+        # https://github.com/WebAssembly/binaryen/issues/3213
+        'instrument-locals_all-features.wast',
+        'remove-unused-names_code-folding_all-features.wast',
+        'Os_print-stack-ir_all-features.wast'
```

Comment:
```suggestion
        'too_much_for_liveness.wasm'
```
After #3282 lands there will be no invalid pops.